recipes/bioconductor-monocle/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = 'monocle' %}
+{% set version = '2.4.0' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url:
+    - https://bioconductor.org/packages/release/bioc/src/contrib/{{ name }}_{{ version }}.tar.gz
+  md5: fef48c57d5da92fae71cb501d8603cf0
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+
+requirements:
+  build:
+    - r-base
+    - r-biobase
```

Comment:
I am sure I'm doing something incorrectly, but can't figure it out. I am running the commands from the anaconda/envs/bioconductor-recipes directory and running 
`python bioconductor_skeleton.py monocle` returns `ImportError: No module named configparser'.


This question/suggestion comes out of frustration, but if there is a script for generating Conda recipes from bioconductor (and CRAN, pypi, etc) why not just have this script sweep through bioconductor to generate a recipe for each tool in bioconductor? This would make it much simpler to wrap these tools for Galaxy.